static/panes/compiler.ts

```diff
@@ -1487,7 +1493,10 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         this.updateDecorations();
 
         const codeLenses: monaco.languages.CodeLens[] = [];
-        if (this.getEffectiveFilters().binary || result.forceBinaryView) {
+        if (this.getEffectiveFilters().binary
```

Comment:
Can you refactor a tiny bit and bring out a constant: calling getEffectiveFilters() twice seems ugly:

```
const effectiveFilters = this.getEffectiveFilters();
if (effectiveFitlers.binary || effectiveFitlers.binaryObject...)
```

type of thing

Reply:
Will do!